src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName, isMarried } = person;
+
+  return (
+    <div className="App">
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+        {age && (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        )}
+
+        <p className="Person__partner">
+          {isMarried
+            ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
move this to the separate variable
```suggestion
            ? `${partnerName} is my ${partner}`
```